Longest_Common_Subsequence/Longest_Common_Subsequence.rb

```diff
@@ -0,0 +1,40 @@
+# Problem Statement- Given two sequences, find 
```

Comment:
Please convert these to a single multi line comment.

Reply:
Converted them to a single multi-line comment